scripts/lib/CIME/case.py

```diff
@@ -1330,8 +1330,6 @@ def set_file(self, xmlfile):
         gfile = GenericXML(infile=xmlfile)
         ftype = gfile.get_id()
 
```

Comment:
This is an unrelated change - It looks like a merge error, the flushalll call is at line 1328.

Reply:
Gee wiz, I thought only subversion did stupid merge stuff like this. Sigh.